migration-guide.adoc

```diff
@@ -76,3 +76,26 @@ Such changes typically do not impact programs using a relational schema managed
 
 This section describes changes to dependencies used by Hibernate ORM.
 
+[[maven-plugin-changes]]
+== Changes to the Maven Plugin
+
+The Maven bytecode enhancement plugin has been completely rewritten since version 7.0. The change that has the
+biggest consequence is that the artefactId and groupId of the Maven artefact has changed. As of now, you will
+need to include the plugin in the build section in your Maven pom file as illustrated below.
+
+```
+<plugin>
+    <groupId>org.hibernate.orm</groupId>
+    <artifactId>hibernate-maven-plugin</artifactId>
+    <version>$currentHibernateVersion</version>
+    <executions>
+        <execution>
+            <goals>
+                <goal>enhance</goal>
+            </goals>
+        </execution>
+    </executions>
+</plugin>
+```
+
+The documentation in the userguide has been rewritten accordingly and completed with more examples.
```

Comment:
This should be merged into 7.0, not into 7.1.

Reply:
Ok, that was also my original setup. I'll remove the change.